dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
Agreed, swagger is helpful for auto-generation for documentation purposes. But what we needed it for was collaboration during planning. I've found some open source tools that may be a helpful alternative. I can dive into those a little more and see if they're a viable solution.